docs/external_services/github_pages.md

```diff
@@ -0,0 +1,42 @@
+# Github Pages
+
+To deploy a documentation site (like this one), we currently use [Github Pages][github-pages]
+
+[Travis](travis.md) can deploy the documentation to github pages.
+
+1. Add the following to the `.travis.yml` file to deploy to github pages. _This is included in the sample file_
```

Comment:
The fragment below doesn't match what's in `.travis.yml`. Do you want what's in the documentation to be just representative or the actual required contents?